projects/aca-about/src/lib/about.component.ts

```diff
@@ -23,75 +23,22 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { ExtensionRef } from '@alfresco/adf-extensions';
-import { Component, OnInit, ViewEncapsulation } from '@angular/core';
-import { RepositoryInfo } from '@alfresco/js-api';
-import { Observable } from 'rxjs';
-import { map } from 'rxjs/operators';
-import { AppExtensionService, ContentApiService } from '@alfresco/aca-shared';
-import { dependencies, version, name, commit } from 'package.json';
+import { Component, Inject } from '@angular/core';
+import pkg from 'package.json';
 
 @Component({
-  selector: 'app-about',
+  selector: 'app-about-page',
   templateUrl: './about.component.html',
-  styleUrls: ['./about.component.scss'],
-  encapsulation: ViewEncapsulation.None,
-  host: { class: 'app-about' }
+  styleUrls: ['./about.component.scss']
 })
-export class AboutComponent implements OnInit {
-  repository: RepositoryInfo;
-  extensions$: Observable<ExtensionRef[]>;
-  dependencyEntries: Array<{ name: string; version: string }>;
-  statusEntries: Array<{ property: string; value: string }>;
-  licenseEntries: Array<{ property: string; value: string }>;
-  adfRepoUrl = 'https://github.com/Alfresco/alfresco-ng2-components/commits';
-  appRepoUrl = !!commit ? `https://github.com/Alfresco/${name}/commits/${commit}` : null;
-  adfVersion = '';
-  appVersion = version;
-
-  constructor(private contentApi: ContentApiService, appExtensions: AppExtensionService) {
-    this.extensions$ = appExtensions.references$;
+export class AboutComponent {
+  pkg: any;
+  dev: boolean = false;
+
+  constructor(@Inject('environment')
+                environment) {
+    this.dev = !environment.production;
```

Comment:
Do we need the whole env object? In the upcoming release, we are about to add the canary flags into the environment objects. Do you need them as well or just the production flag?